liquibase-service/src/main/resources/db/rdb/tables/017-create_nrt_observation-003.sql

```diff
@@ -0,0 +1,249 @@
+IF EXISTS (SELECT 1 FROM sysobjects WHERE name = 'nrt_observation' and xtype = 'U')
+    BEGIN
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'ctrl_cd_display_form' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+                ALTER TABLE nrt_observation
+                    ADD ctrl_cd_display_form varchar(20);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'obs_domain_cd_st_1' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+                ALTER TABLE nrt_observation
+                    ADD obs_domain_cd_st_1 varchar(20);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'processing_decision_cd' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+                ALTER TABLE nrt_observation
+                    ADD processing_decision_cd varchar(20);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'cd' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+                ALTER TABLE nrt_observation
+                    ADD cd varchar(50);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'shared_ind' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+                ALTER TABLE nrt_observation
+                    ADD shared_ind char(1);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'status_cd' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+            ALTER TABLE nrt_observation
+                ADD status_cd char(1);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'cd_system_cd' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+            ALTER TABLE nrt_observation
+                ADD cd_system_cd varchar(50);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'cd_system_desc_txt' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+            ALTER TABLE nrt_observation
+                ADD cd_system_desc_txt varchar(1000);
```

Comment:
Is this sufficient. Usually QA test for a large data.

Reply:
Good id selection- The varchar value for ODSE's Observation's cd_system_desc_txt is 100. This should be 100, not 1000. 
That being said, the stored procedure can be updated by enforcing a string limit in the pre-processing.

Reply:
It's more than sufficient, it's actually varchar(100) in ODSE Observation.